tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -71,6 +71,7 @@ def run_conv(
     groups=1,
     has_bias=True,
     shard_layout=None,
+    use_max_cores=False,
```

Comment:
what is this parameter for?

Reply:
Using this variable we are removing the restriction of shard shape being in multiple of TILE_SIZE. Without this, there will have issues in networks since output shard shape is getting changed now.